src/Read/ParserBase.cpp

```diff
@@ -127,6 +127,13 @@ void ParserBase::GotChunk( const ChunkHeader& ch )
       //_StreamInfo.back().type2index.Read( _Stream, ch.EndPos() );
    }
 
+   // Type 2 AVIs may have an 'idx1' chunk but we don't want those... hence the check on indxChunk.size
+   if ( ch.fcc == FCC( 'idx1' ) && !_StreamInfo.empty() && _StreamInfo.back().indxChunk.size == 0 )
+   {
+      _StreamInfo.back().indxChunk = ch;
+      _StreamInfo.back().isTypeOneAVI = true;
+   }
+
```

Comment:
i'm confused by this scenario

Reply:
For some of the "type 2" AVI files I was looking at, there would be both an 'indx' and 'idx1' chunk. My understanding of the file format is limited but I think the 'idx1' chunk should be ignored in that case?